### DISCUSSION TITLE
StockDiv, Your portfolio. Your Income

### DISCUSSION THREAD

**Discussion by AmosSzust** [quasarframework/quasar#17006] [Show and Tell]
Made with Quasar (and node)
https://www.stockdiv.com
https://www.stockdiv.com/login
![screenshot](https://github.com/quasarframework/quasar/assets/100950035/7521d3ae-1e0f-407a-bd9c-5a82ff8afbe4)

**Comment by etewiah** [comment]
Looks good, thanks for sharing.
